ush/config_defaults.yaml

```diff
@@ -1746,17 +1781,19 @@ task_make_lbcs:
   # This file should be the same in both make_ics and make_lbcs.
   #
   #------------------------------------------------------------------------
-  KMP_AFFINITY_MAKE_LBCS: "scatter"
-  OMP_NUM_THREADS_MAKE_LBCS: 1
-  OMP_STACKSIZE_MAKE_LBCS: "1024m"
-  VCOORD_FILE: "{{ workflow.FIXam }}/global_hyblev.l65.txt"
+  envars:
+    KMP_AFFINITY_MAKE_LBCS: "scatter"
+    OMP_NUM_THREADS_MAKE_LBCS: 1
+    OMP_STACKSIZE_MAKE_LBCS: "1024m"
+    VCOORD_FILE: "{{ workflow.FIXam }}/global_hyblev.l65.txt"
 
 #----------------------------
 # IO_LAYOUT_Y FORECAST config parameters
 #-----------------------------
 task_run_fcst:
   # UW Placeholder additions
   rundir: '{{ workflow.EXPTDIR }}/{{ timevars.yyyymmddhh }}{{ "/mem%s" % ("MEMBER"|env) if global.DO_ENSEMBLE }}'
+  walltime: 04:30:00
```

Comment:
I was lazy here, and this one will be subsumed by work @NaureenBharwaniNOAA will be contributing.